ngraph/core/builder/include/ngraph/builder/make_constant.hpp

```diff
@@ -116,5 +116,18 @@ namespace ngraph
 
             return val->add_provenance_group_members_above({});
         }
+
+        /// \brief      Create constant filled with double value
+        ///
+        /// \note       If num value exeeds capacity of type, the value is clamped.
+        ///
+        /// \param[in]  type           The type of produced Constant node.
+        /// \param[in]  shape          The shape of produced Constant node.
+        /// \param[in]  num            The value used to fill Constant node.
+        ///
+        /// \return     The Constant node which have expected type, shape and value.
+        ///
+        std::shared_ptr<Node>
+            make_constant_from_double(const element::Type& type, const Shape& shape, double num);
```

Comment:
Is it needed for ONNX? Because f64 is not used inside the IE and I don't know places where we work with double.

Reply:
Creation constants from double was implemented before in Clamp decompose method (Clamp takes double attributes).
I just move it to builder (as helper) because it is useful to create Constant with max/min value of any type (it provides correct double to int conversion with checking range)